src/shared/extensionGlobals.ts

```diff
@@ -81,7 +81,7 @@ export function checkDidReload(context: ExtensionContext): boolean {
     return !!context.globalState.get<string>('ACTIVATION_LAUNCH_PATH_KEY')
 }
 
-export function initialize(context: ExtensionContext): ToolkitGlobals {
+export async function initialize(context: ExtensionContext): Promise<ToolkitGlobals> {
```

Comment:
Does this need to be async?

Reply:
Not anymore, I was awaiting the command in init in a past version